packages/user-management-widget/src/lib/widget/api/sdk/createUserSdk.ts

```diff
@@ -140,11 +140,22 @@ export const createUserSdk = ({
     return res.json();
   };
 
+  const getTenantRoles = async () => {
```

Comment:
Also this one, need to put it under a different name

Reply:
Fixed